edb/server/protocol/binary.pyx

```diff
@@ -499,6 +499,27 @@ cdef class EdgeConnection(frontend.FrontendConnection):
         else:
             return edgeql.NormalizedSource.from_string(text)
 
+    async def _suppress_tx_timeout(self):
+        async with self.with_pgcon() as conn:
+            await conn.sql_execute(b'''
+                select pg_catalog.set_config(
+                    'idle_in_transaction_session_timeout', '0', false)
```

Comment:
You probably want the opposite?

```suggestion
                    'idle_in_transaction_session_timeout', '0', true)
```

Reply:
Sure. I don't think it should matter in practice, since there shouldn't be any way for this set_config to ever get committed